data/mods/Aftershock/spells/enchantments.json

```diff
@@ -28,5 +28,58 @@
     "id": "protect_cold",
     "condition": "ACTIVE",
     "values": [ { "value": "ARMOR_COLD", "add": -15 } ]
+  },
+  {
+    "type": "enchantment",
+    "id": "forcefield_bash_cut_weak",
+    "condition": "ACTIVE",
+    "values": [ { "value": "ARMOR_BASH", "add": -5 }, { "value": "ARMOR_CUT", "add": -5 } ]
+  },
+  {
+    "type": "enchantment",
+    "id": "forcefield_bash_cut_medium",
+    "condition": "ACTIVE",
+    "values": [ { "value": "ARMOR_BASH", "add": -10 }, { "value": "ARMOR_CUT", "add": -10 } ]
+  },
+  {
+    "type": "enchantment",
+    "id": "forcefield_bash_cut_heavy",
+    "condition": "ACTIVE",
+    "values": [ { "value": "ARMOR_BASH", "add": -15 }, { "value": "ARMOR_CUT", "add": -15 } ]
```

Comment:
The value for these enchantments are a bit close together. I think you could go 5/15/25 instead